db/schema.rb

```diff
@@ -175,6 +174,12 @@
     t.index ["brand_id"], name: "index_twitter_trackers_on_brand_id", unique: true, using: :btree
   end
 
+  create_table "user", force: :cascade do |t|
```

Comment:
gotcha, no clue why 'user' was there but it's not even in my file local repo anymore. strange.